src/App.js

```diff
@@ -1,11 +1,69 @@
 import React, { Component } from 'react';
+import {Grid, Row} from 'react-bootstrap';
+import axios from 'axios';
+
 import './App.css';
+import NavBar from './NavBar';
+import JumboContent from './JumboContent';
+import SideNav from './SideNav';
+import Footer from './Footer';
+import ModalForm from './ModalForm';
+
 
+/* eslint-disable max-len */
 class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      jumbotronHeader: 'Hello, World!',
+      jumbotronContent: 'This is an example to show the potential of an offcanvas layout pattern in Bootstrap. Try some responsive-range viewport sizes to see it in action.',
+      teasers: []
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://www.localhost:4000/teasers')
+      .then(resp => {
+        this.setState({
+          teasers: resp.data
+        });
+      })
+      .catch(err => {
+        console.log(`Error ${err}`);
+      });
+  }
+
+  handleContentSubmit(attributes) {
```

Comment:
now if we console.log whats being passed to our function in app, we'll see title is being passed properly.